src/core/Tools.h

```diff
@@ -41,8 +41,10 @@ namespace Tools
     void sleep(int ms);
     void wait(int ms);
     QString uuidToHex(const QUuid& uuid);
-    QRegularExpression convertToRegex(const QString& string, bool useWildcards = false,
-                                      bool exactMatch = false, bool caseSensitive = false);
+    QRegularExpression convertToRegex(const QString& string,
+                                      bool useWildcards = false,
+                                      bool exactMatch = false,
+                                      bool caseSensitive = false);
```

Comment:
I didn't change this line on purpose. I've set Clang-Format to run on save and that line didn't comply with the rules. I didn't want to make a "Formatting" commit for such a small change.

Reply:
Agree, it is also changed in my project-wide formatting PR.